### PR TITLE
Application Status History

### PR DESCRIPTION
**Pull request by srleach**
## Features

- Add 'Get Application History' method to `ApplicationGateway`

Delivers [#126577079](https://www.pivotaltracker.com/story/show/126577079)

### CODE REVIEW

File: src/Gateways/ApplicationGateway.php [hunk 1]
```diff
@@ -170,6 +170,24 @@ public function getApplicationCreditInfo($installationId, $application, $token)
...
+     * @param $token
+     * @return array
+     */
+    public function getApplicationHistory($application, $token)
+    {
+        return $this->fetchDocument(
+            '/v4/applications/' . $application . '/status-history',
+            $token,
+            'Application Status History',
+            []
```
Comment (wnowicki) [inline src/Gateways/ApplicationGateway.php]: - last param is optional, suggesting to use default value

**Comment by wnowicki** [comment]
@srleach

File: src/Gateways/ApplicationGateway.php (hunk 1 above)
Comment (srleach) [inline src/Gateways/ApplicationGateway.php]: 👍

**Comment by srleach** [comment]
@wnowicki

**Comment by wnowicki** [comment]
👍 waiting for CI

**Comment by wnowicki** [comment]
- tests are missing, please add them
- quality fine

@srleach

**Comment by srleach** [comment]
@wnowicki

**Comment by wnowicki** [comment]
- 👍 code
- 👍 quality
- 👍 tests
- 💯 tests improved

**Pull request merged**